### ISSUE TITLE
dataset division

### ISSUE THREAD

**Issue by zhangyingerjelly** [Waller-Lab/LenslessLearning#5]
I want to do a comparision to the result of le-admm-unet, but I don’t know division of training set and test set in the dataset for the given trained model.
